src/uu/seq/src/seq.rs

```diff
@@ -47,9 +48,33 @@ struct SeqOptions<'a> {
 /// The elements are (first, increment, last).
 type RangeFloat = (ExtendedBigDecimal, ExtendedBigDecimal, ExtendedBigDecimal);
 
+// Turn short args with attached value, for example "-s,", into two args "-s" and "," to make
+// them work with clap.
```

Comment:
maybe add a link to the discussion ?

Reply:
Yes, good point, though I will add it to the comment on lines 196/197 where it's a better fit.